src/main/java/ConsoleInterface.java

```diff
@@ -0,0 +1,201 @@
+import java.util.Scanner;
+
+public class ConsoleInterface {
+    private static final String COMMAND_BYE = "bye";
+    private static final String COMMAND_LIST = "list";
+    private static final String COMMAND_MARK = "mark";
+    private static final String COMMAND_UNMARK = "unmark";
+    private static final String COMMAND_TODO = "todo";
+    private static final String COMMAND_DEADLINE = "deadline";
+    private static final String COMMAND_EVENT = "event";
```

Comment:
Great! You have followed the correct naming convention

Reply:
Using constants for commands is a good idea